pkg/cmd/root.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/nitrictech/cli/pkg/output"
 )
 
+const feedbackMsg = "Thanks for trying nitric!\nIf you have feedback you can raise issues on GitHub https://github.com/nitrictech/nitric or come talk with us directly on Discord https://discord.gg/EBPCEgG9"
```

Comment:
Not sure how we can sort this out, the current invite link contains your username for discord. This isn't a huge issue I think, but good to see if we could get around this.

Reply:
@tjholm @HomelessDinosaur how's this link? https://discord.gg/bvTW7aYg6B

Reply:
@tjholm maybe we just make a nitric user in discord for this purpose.

Reply:
Created a new invite link with my current user `tjh`: https://discord.com/invite/Webemece5C

Reply:
Will look at server boosting later to get a custom invite link going.

Reply:
Change to https://discord.com/invite/Webemece5C, as posted on twitter